src/index.html

```diff
@@ -1,13 +1,45 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body class="body">
+    <div class="card" data-qa="card">
+      <img
+        src="./images/imac.jpeg"
+        alt="imac"
+        class="card_img"
+      >
+      <h2 class="card_name">
+          APPLE A1419 iMac 27" Retina <br>5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="card_code">Product code: 195434</p>
+      <div class="split_blocks">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card_reviews-text">Reviews: 5 </p>
+      </div>
+      <p class="card_price-text">Price: <strong class="card_price">$2,199</strong></p>
+      <a
+          href="#"
+          class="card_link"
+          data-qa="hover"
+      >
```

Comment:
Pay attention on indents. it should be 2 spaces
```suggestion
      <a
          href="#"
          class="card_link"
          data-qa="hover"
      >
```